Sources/SwiftSyntax/Raw/RawSyntax.swift

```diff
@@ -474,12 +474,12 @@ extension RawSyntax {
     }
   }
 
-  func formLeadingTrivia() -> Trivia? {
-    firstToken(viewMode: .sourceAccurate)?.formLeadingTrivia()
+  func formLeadingTrivia() -> Trivia {
+    firstToken(viewMode: .sourceAccurate)?.formLeadingTrivia() ?? Trivia.zero
```

Comment:
Not sure if this is the consensus in this repo, but will ` ?? .zero` be possible?

Same below?

Reply:
@bnbarham is making `Trivia` non-optional in his next PR 😉  https://github.com/apple/swift-syntax/pull/1478

Reply:
I'm not quite sure what you mean here, this *is* the non-optional trivia PR 😅. @kimdv are you just asking if we *should* have a `.zero` or whether this case ever actually happens?

Reply:
Nevermind, I was stupid. I thought the optionality was coming from `formLeadingTrivia()` instead of the optional chaining.

Reply:
I was meaning `.zero` is that possible 😅
Sorry for confusing comment